tests/unit/test_in_memory_database.py

```diff
@@ -0,0 +1,13 @@
+"""
+Copyright (c) 2024 Aiven Ltd
+See LICENSE for details
+"""
+from karapace.in_memory_database import InMemoryDatabase, Subject
+
+
+def test_find_schemas() -> None:
```

Comment:
issue: As this test has a very specific purpose: verify we don't blow up when given an empty string, I think we should have more specific naming of this test. I find this to be the best way to structure test for maximizing communication of original intent:

```python
class TestFindSchemas:  # <- _what_ are we testing
    def test_returns_empty_list_when_no_schemas(  # <- what _expectation_ do we have on the tested thing
        self,
    ) -> None:
        ... 
    # Future tests can easily be added here:
    def test_raises_foo_error_when_stars_are_not_aligned() -> None: ...
```